server.go

```diff
@@ -65,9 +61,7 @@ var (
 
 // NewServer creates a new Mixer metric handling server.
 func NewServer(addr string, h *telemetry.Harvester, grpcOpt ...grpc.ServerOption) (*Server, error) {
-	gp := pool.NewGoroutinePool(5, false)
```

Comment:
Looks like this was never actually used before, other than to create a `NewEnv`, is that correct?

Reply:
Correct.